### PR TITLE
Windows server support update

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/K6JDIYpt/393-dropping-windows and gdoc version at https://docs.google.com/document/d/1hjWf7-HvN0bVzMCA8stBAmKtGcNIr9Xh4zd02lGH7BY/edit#

There is no staging build for the blog so you need to fetch the branch and build locally with Jekyll if you want to review the formatting, however, I've already checked that and this really only needs proof-reading before going live.

### CODE REVIEW

File: _posts/2016-03-21-windows-support.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Windows support update
+categories: tech-issues future-plans
```
Comment (joshmoore) [inline _posts/2016-03-21-windows-support.md]: We don't have anything like it yet but would "deployment" be better than "tech-issues"?

File: _posts/2016-03-21-windows-support.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+The OME team has always been committed to building specifications and software
+that are cross-platform solutions and that support as many different types of
+users as possible, including those with limited IT support/budget. In keeping
+with our open source ethos and the fact we rely on public and charity grant
+funding, we also try to use open source tools as part of our workflows for
+testing, building and deploying our products wherever possible.  We do support
+commercial OS and platforms — we build and test OMERO.insight on Windows,
```
Comment (manics) [inline _posts/2016-03-21-windows-support.md]: `We do support commercial OS and platform`. Operating systems? Or open-source?

File: _posts/2016-03-21-windows-support.md (hunk 1 above)
Comment (hflynn) [inline _posts/2016-03-21-windows-support.md]: I used these categories as they are the same as we used to java webstart support. I can add deployment if you like.

File: _posts/2016-03-21-windows-support.md (hunk 2 above)
Comment (hflynn) [inline _posts/2016-03-21-windows-support.md]: The former, I'll clarify

File: _posts/2016-03-21-windows-support.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+We improved our server installation guides and OMERO.web deployment
+documentation and provided stepwise deployment scripts, e.g. for CentOS 6 with
+Python 2.7. We extensively used [Docker](https://www.docker.com/) to test our
+Linux installation scripts and also to test our installation documentation.
+All the installation scripts are available (see
+[omero-install](https://github.com/ome/omero-install/)) and run on the CI
+system each time a commit is pushed.
+
+During the development phase of OMERO 5.2.1, we dedicated a large amount of
+extra resources—devops team and cores—to test the Windows installation
```
Comment (manics) [inline _posts/2016-03-21-windows-support.md]: `devops team and cores`- this phrase isn't clear

File: _posts/2016-03-21-windows-support.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+with our open source ethos and the fact we rely on public and charity grant
+funding, we also try to use open source tools as part of our workflows for
+testing, building and deploying our products wherever possible.  We do support
+commercial OS and platforms — we build and test OMERO.insight on Windows,
+actively work to support accessing the OMERO and Bio-Formats APIs in Matlab,
+and actively support browsing OMERO.web using IE.
+
+Since the beginning of the OMERO project, we have actively tested and
+supported builds and tests of OMERO.server on Windows. Several users—
+students, faculty and institutions—have highlighted the importance of this
```
Comment (will-moore) [inline _posts/2016-03-21-windows-support.md]: users-students, institutions-have  Are these dashes meant to be there?

File: _posts/2016-03-21-windows-support.md (hunk 3 above)
Comment (hflynn) [inline _posts/2016-03-21-windows-support.md]: That's Jason, I'll reword it

File: _posts/2016-03-21-windows-support.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+matrix is constantly growing; already we are adding Ice 3.6 and soon Java 1.9
+will need to be on our radar too. The resources we have for building and
+maintaining our CI system aren’t infinite. We have to balance these resources
+with core development work. There’s always a tension between rapid development
+of new functionality and robust, reliable testing.
+
+The focus of our OMERO 5.2.1 release was on deployment, following feedback
+from system administrators (e.g. [this forum thread](https://www.openmicroscopy.org/community/viewtopic.php?f=5&t=7924)).
+We improved our server installation guides and OMERO.web deployment
+documentation and provided stepwise deployment scripts, e.g. for CentOS 6 with
```
Comment (manics) [inline _posts/2016-03-21-windows-support.md]: Two `and`s in the same sentence

File: _posts/2016-03-21-windows-support.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+so this decision should have relatively limited impact. We are, nonetheless,
+an open source project and aim to support our incredibly diverse community and
+the needs they have to deploy our software. So, even if only a few
+OMERO.server installations run on Windows, we very much want to support them.
+We simply can’t afford to do so.
+
+For all these reasons, from version 5.3, we will not be able to support
+OMERO.server on Windows. Again, this is a build and testing issue, so if
+anyone out there would like to contribute their time, energy, expertise and
+cores to provide Windows support, we’d welcome them doing so. Instead, we will
```
Comment (manics) [inline _posts/2016-03-21-windows-support.md]: I guess `cores` = `compute resources`?

File: _posts/2016-03-21-windows-support.md (hunk 4 above)
Comment (hflynn) [inline _posts/2016-03-21-windows-support.md]: Yes, it's using em dashes to replace parentheses

Comment (will-moore) [inline _posts/2016-03-21-windows-support.md]: Oh, I see you've used them elsewhere as brackets. So it's meant to be read like "Several users (students, faculty and institutions) have highlighted...". I've never seen dashes used like that before.

Comment (hflynn) [inline _posts/2016-03-21-windows-support.md]: It's just a style thing really, Jason is rather fond of them ;)

**Comment by hflynn** [comment]
@manics better now?

**Comment by jburel** [comment]
Thanks for the update @hflynn, read well.
+1

**Pull request merged**